connect/cli/plugins/product/export.py

```diff
@@ -873,6 +899,49 @@ def _dump_items(ws, client, product_id, silent):
     print()
 
 
+def _dump_translations(ws, client, product_id, silent):
+    _setup_ws_header(ws, 'translations')
+    ws.column_dimensions['F'].width = 30
+    ws.column_dimensions['J'].width = 15
+    ws.column_dimensions['K'].width = 15
+    row_idx = 2
+
+    rql = R().context.instance_id.eq(product_id)
+
+    translations = (
+        client.ns('localization')
+        .translations
+        .filter(rql)
+    )
+    count = translations.count()
+
+    action_validation = DataValidation(
+        type='list',
+        formula1='"-,delete,update,create"',
+        allow_blank=False,
+    )
+    disabled_enabled = DataValidation(
+        type='list',
+        formula1='"Disabled,Enabled"',
+        allow_blank=False,
+    )
+    ws.add_data_validation(action_validation)
+    ws.add_data_validation(disabled_enabled)
+
+    progress = trange(0, count, disable=silent, leave=True, bar_format=DEFAULT_BAR_FORMAT)
+
+    for translation in translations:
+        progress.set_description(f'Processing translation {translation["id"]}')
+        progress.update(1)
+        _fill_translation_row(ws, row_idx, translation)
+        action_validation.add(ws[f'B{row_idx}'])
+        disabled_enabled.add(ws[f'I{row_idx}'])
+        row_idx += 1
```

Comment:
Wow, that's confusing a little. I would propose to have something like

```
for index, translation in enumerate(translations):
   ...
   sheet_index = index + 2
    _fill_translation_row(ws, sheet_index, translation)
```

Wdyt?

Reply:
Yes it could be, or maybe:
```
for row_idx, translation in enumerate(translations, 2):
         ....
      _fill_translation_row(ws, row_idx, translation)
```
So i do not have to increment row index, and delegates it to `enumerate`